components/list/test/list.test.js

```diff
@@ -117,22 +114,22 @@ describe('d2l-list', () => {
 
 				const listItem = elem.querySelector('[key="L2-2"]');
 				const listItemLayout = elem.querySelector('[key="L2-2"]').shadowRoot.querySelector('d2l-list-item-generic-layout');
-				listItem.focus();
+				await focusWithKeyboard(listItem);
 				await waitUntil(() => listItem.hasAttribute('_focusing'), 'List item should be focused', { timeout: 3000 });
 
-				const event = new KeyboardEvent('keydown', { keyCode: testCase.keyPress.key });
-				setTimeout(() => listItemLayout.dispatchEvent(event));
+				await focusWithKeyboard(listItemLayout);
```

Comment:
Why do we need to focus `listItem` and `listItemLayout` separately now?

Reply:
It has to do with how `sendKeys` works compared to our simulation previously. `sendKeys` itself doesn't take an element, so it'll send those key presses to whatever element currently has focus. Previously we were focusing on the list item to put it into "focus" mode, and they simulating a key press on the layout element.